network/src/peer_manager/mod.rs

```diff
@@ -985,10 +985,11 @@ where
             }
             Err(error) => {
                 error!(
-                    "{} Error dialing Peer {} at {}",
+                    "{} Error dialing Peer {} at {}: {}",
                     self.network_context,
                     peer_id.short_str(),
-                    addr
+                    addr,
+                    error.to_string()
```

Comment:
i don't think we need the `.to_string()`, since `.to_string()` just uses the Display impl anyway, so we can remove it

can also avoid an extra allocation : p

Reply:
putting an error in the log, will make it both do "display" and "debug" in the structured logs :(